sidebars.js

```diff
@@ -46,17 +46,17 @@ module.exports = {
     },
     {
       type: "category",
-      label: "GraphQL",
+      label: "API Usage",
       link: {
-        type: "doc",
-        id: "developer/getting-started/graphql",
+        type: "generated-index",
+        title: "API Usage",
+        description: "Saleor API conventions.",
```

Comment:
I think this description text could be better, conventions?

Reply:
Go ahead and improve it.